src/Store.ts

```diff
@@ -96,17 +96,26 @@ export abstract class Store extends Dispatcher implements StoreLike {
     }
 
     /**
+     * ## Write phase in read-side
+     *
+     * You can update own state.
+     */
+    receivePayload?(payload: Payload): void;
```

Comment:
Add `Store#receivePayload`.
We want to make it more meaning name.

It is write phase in read-stack